optional_plugins/loader_yaml/avocado_loader_yaml/__init__.py

```diff
@@ -134,16 +135,29 @@ class LoaderYAML(CLI):
     description = "YAML test loader options for the 'run' subcommand"
 
     def configure(self, parser):
-        for name in ("list", "run"):
```

Comment:
The support for `list` is still valid, and so it should remain.

This actually brings the question of options that may apply at a broader level, or in a more specific level. For instance, a user may want to have `mux_suite_only` applied to *both* `list` and `run`. How should we approach this? The easy answer is to require users to provide both configurations, but maybe we could have a shortcut, such as:

```json
config = {"*.mux_suite_only": "/path/to/filter"}
```

Just brainstorming though.

Reply:
@clebergnu Besides the `for` loop, currently `list` is not working with those parameters. See the output of `avocado list -h | grep mux`. Should we add it now?

About the `*`: I have the preference to not introduce the `*` option, at least for now. If something is on a broader level, I have the feeling that is better if we add them to the "core" or global (like `--version`, `--config`, `--paginator`, `--show`, etc.) It will keep the things simple. What do you think?